.buildkite/release.rayci.yml

```diff
@@ -8,7 +8,7 @@ steps:
       - skip-on-premerge
       - oss
     commands:
-      - ./release/run_release_test.sh microbenchmark.aws --report 
+      - bazel run //release:run_release_test -- microbenchmark.aws --report 
```

Comment:
this will by pass a lot of logic in the shell script `run_release_test.sh`?

Reply:
Yes, let me keep the shell script for ray for now; the shell script has the following additional logic:
- copy release outputs to artifacts
- retries

Both are non-critical; i mainly do this so i can reuse the release test framework on llvm-forge which doesn't require those two logics. Overtime, I can move those two logics into the py_binary itself to make the stack simpler.

Reply:
the script also handles cancellation on sigterm/sigint?

can we call bazel on this `py_binary` in the shell script too?

Reply:
yes let me do that; i do want to get rid of this shell script layer, but I can do that in a follow up